packages/webdriverio/src/utils/interception/index.ts

```diff
@@ -1,7 +1,32 @@
+/// <reference types="webdriverio/webdriverio"/>
 import Timer from '../Timer'
 
+type FilterOptions = {
+    method?: string;
+    header?: object;
+    responseHeaders?: object;
+};
+
+type WaitForResponseFn = {
+    timeout?: number;
+    interval?: number;
+    timeoutMsg?: string;
+}
+
 export default class Interception {
-    constructor (url, filterOptions = {}, browser) {
+    url: string;
+    filterOptions: FilterOptions;
+    browser: WebdriverIO.Browser;
+    respondOverwrites: {
+        overwrite: any;
```

Comment:
this needs a type definition

Reply:
yes, I know I know! Just opened as a draft because I wanted to know how to structure it before going deeper into the types! So I guess it's ok to use `<reference types="webdriverio/webdriverio"/>` here?

Reply:
I assumed it's fine, updated! :)